include/gunrock/framework/operators/advance/block_mapped.hxx

```diff
@@ -196,8 +196,9 @@ void execute(graph_t& G,
       typename work_tiles_t::value_type,  // segments value type
       operator_t                          // lambda type
       >;
-  launch_box.launch(__bm, G, op, input->data(), output->data(), num_elements,
-                    segments.data().get());
+  auto __args = std::make_tuple(G, op, input->data(), output->data(),
+                                num_elements, segments.data().get());
+  launch_box.launch(__bm, __args, context);
```

Comment:
@neoblizz here is how the usage can be done now by packing tuples into arguments. You can see here we can declare a tuple and then pass it in when there are a lot of arguments.